backend/inspirehep/records/models.py

```diff
@@ -262,3 +263,49 @@ class StudentsAdvisors(db.Model):
     degree_type = db.Column(
         ENUM(*[key.value for key in DegreeType], name="enum_degree_type")
     )
+
+
+class Timestamp(object):
+    """Timestamp model mix-in with fractional seconds support.
+    SQLAlchemy-Utils timestamp model does not have support for fractional
+    seconds.
+    """
+
+    created = db.Column(
+        db.DateTime(),
+        default=datetime.utcnow,
+        nullable=True,
+    )
+    updated = db.Column(
+        db.DateTime(),
+        default=datetime.utcnow,
+        nullable=True,
+    )
```

Comment:
No need to define it again `from sqlalchemy_utils.models import Timestamp`

Reply:
yes, I wanted to do that but `SQLAlchemy-Utils timestamp model does not have support for fractional seconds`